labgrid/driver/commandmixin.py

```diff
@@ -34,14 +34,14 @@ def _run_check(self, cmd: str, timeout=30):
         Returns:
             List[str]: stdout of the executed command
         """
-        stdout, stderr, exitcode = self._run(cmd, timeout=timeout)
+        stdout, stderr, exitcode = self._run(cmd, timeout=timeout, codec=codec, decodeerrors=decodeerrors)
```

Comment:
Where are `codec` and `decodeerrors` supposed to come from? I guess you confused different methods here.